src/Component/Tests/Unit/Filtering/FiltersCriteriaResolverTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Tests\Unit\Filtering;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Grid\Definition\Filter;
+use Sylius\Component\Grid\Definition\Grid;
+use Sylius\Component\Grid\Filtering\FiltersCriteriaResolver;
+use Sylius\Component\Grid\Filtering\FiltersCriteriaResolverInterface;
+use Sylius\Component\Grid\Parameters;
+
+final class FiltersCriteriaResolverTest extends TestCase
+{
+    private FiltersCriteriaResolver $filtersCriteriaResolver;
+
+    protected function setUp(): void
+    {
+        $this->filtersCriteriaResolver = new FiltersCriteriaResolver();
+    }
+
+    public function testItImplementsFiltersCriteriaResolverInterface(): void
```

Comment:
```suggestion
    public function testImplementsFiltersCriteriaResolverInterface(): void
```
For sake of consistency